### ISSUE TITLE
pan3d-viewer error: Could not create shader object (linux)

### ISSUE THREAD

**Issue by johnkit** [Kitware/pan3d#50]
I am observing the same error with 2 linux machines, which both have low-end GPUs (NVIDIA T600 and an old Quadro K620).

To reproduce:
1. Start `pan3d-viewer`
2. Select, for example, the `Xarray examples - air temperature` dataset.
3. Click the `Apply & Render (5KB)` button.
4. In response, the display area turns black and lots of messages get dumped to the terminal. The most significant seem to be

```
ERROR:root:1: #version 150
2024-01-11 11:44:42.189 (  80.923s) [    7F97F0135700]   vtkShaderProgram.cxx:428    ERR| vtkShaderProgram (0x7f9778509440): Could not create shader object.
ERROR:root:Could not create shader object.
```

The attached file has all of the terminal messages, which seem to include source code and some unicode chars. [viewer.log](https://github.com/Kitware/pan3d/files/13906033/viewer.log)

Also fwiw, I am using the Firefox browser.

**Comment by jourdain** [comment]
Does ParaView works on those machines?

**Comment by johnkit** [comment]
Yes, ParaView seems to work ok. here's a screenshot I just took
![wavelet](https://github.com/Kitware/pan3d/assets/5559383/769fbb69-1adf-4c26-aabd-06d35f144bf4)

**Comment by johnkit** [comment]
As another sanity check, the system can also display the Xarray air-temp example outside the browser (using the script in the [pyvista-xarray repo](https://github.com/pyvista/pyvista-xarray) ). 

![image](https://github.com/Kitware/pan3d/assets/5559383/a8c2ab69-28d8-439e-83fa-1cbccd5be731)

**Comment by annehaley** [comment]
Resolved by #59: https://github.com/Kitware/pan3d/pull/59#issuecomment-1932591389
